examples/authorize/src/main/java/com/dropbox/core/examples/authorize/ScopeAuthorize.java

```diff
@@ -17,15 +17,15 @@
 
 /**
  * You must have an api app migrated to new permission to run this example. You app must have
- * scopes "account_info.read", "files.metadata.write", "files.content.read" and
- * "files.content.write" to run this example. You can manage your app's scopes inside your app
- * console's permisison tab.
+ * scopes "account_info.read", "files.metadata.write", "files.content.read",
```

Comment:
Hm, changes to this file were already merged in https://github.com/dropbox/dropbox-sdk-java/pull/315/commits/67070823955219c5c4b78a0bd41b688d3bd6180b but it's showing up again here